package.json

```diff
@@ -51,8 +51,7 @@
     }
   },
   "files": [
-    "dist",
-    "*.d.ts"
+    "dist"
```

Comment:
I am a bit confused 👀

Reply:
Hmmm seems there's no d.ts file on the root level directory, so no difference.

Reply:
Yeah but why `types` are pointing to `index.d.ts`?

Reply:
Would this work? https://antfu.me/notes#types-for-sub-modules

Reply:
Hmm  I think so, but we should add "types" for each entry point. Since you have added `typesVersions`, so this PR will be more simple.